src/apps/service/apps/jobs/submissions.clj

```diff
@@ -45,7 +45,7 @@
   [user input-paths-by-id]
   (->> (flatten (vals input-paths-by-id))
        (remove string/blank?)
-       (get-file-stats user)
+       (#(get-file-stats user % :filter-include "infoType,path,file-size"))
```

Comment:
These filters look correct to me.
Since this is the only spot where this private `get-file-stats` function is used, the `filter-include` list could be pushed into the private `get-file-stats` function, then this line wouldn't have to change.